src/settings.ts

```diff
@@ -18,31 +18,42 @@ const SONARQUBE_CONNECTIONS_CATEGORY = `${SONARLINT_CATEGORY}.${CONNECTIONS_SECT
 const SONARCLOUD_CONNECTIONS_CATEGORY = `${SONARLINT_CATEGORY}.${CONNECTIONS_SECTION}.${SONARCLOUD}`;
 
 async function hasUnmigratedConnections(sqConnections: SonarQubeConnection[],
+                                        scConnections: SonarCloudConnection[],
                                         settingsService: ConnectionSettingsService): Promise<boolean> {
   for (const connection of sqConnections) {
     if (!await settingsService.hasTokenForServer(connection.serverUrl) && connection.token) {
       return true;
     }
   }
+  for (const connection of scConnections) {
+    if (!await settingsService.hasTokenForServer(connection.organizationKey)) {
```

Comment:
Connection should have token, otherwise there is no point to migrate.

Reply:
Yes thanks, I saw it after the rebase.